test/unit/v1.0.0/converter-v1-to-v2.test.js

```diff
@@ -166,4 +240,98 @@ describe('v1.0.0 to v2.0.0', function () {
             });
         });
     });
+
+    describe('scripts', function () {
+        it('should override events with legacy properties if they exist', function (done) {
+            var options = {
+                    inputVersion: '1.0.0',
+                    outputVersion: '2.0.0',
+                    retainIds: true
+                },
+                source = {
+                    preRequestScript: 'console.log("Request level pre-request script");',
+                    tests: 'console.log("Request level test script");',
+                    events: [{
+                        listen: 'prerequest',
+                        script: {
+                            type: 'text/javascript',
+                            exec: ['console.log("Alternative request level pre-request script");']
+                        }
+                    }, {
+                        listen: 'test',
+                        script: {
+                            type: 'text/javascript',
+                            exec: ['console.log("Alternative request level test script");']
+                        }
+                    }]
+                };
+
+            transformer.convertSingle(source, options, function (err, converted) {
+                expect(err).to.not.be.ok;
+
+                // remove `undefined` properties for testing
+                converted = JSON.parse(JSON.stringify(converted));
+
+                expect(converted.event).to.eql([{
```

Comment:
Should assert for legacy properties not present

Reply:
This is a transformation to v2, so they will not be present anyway.

Reply:
they won't be present, shouldn't we assert that is the behavior?